index.ts

```diff
@@ -26,16 +34,16 @@ if (require('semver').satisfies(process.version, '<8') ||
 }
 
 var path = require('path');
-var cls = require('./src/cls.js');
+var cls = require('./src/cls'/*.js*/);
 var common = require('@google-cloud/common');
 var extend = require('extend');
-var constants = require('./src/constants.js');
-var traceUtil = require('./src/util.js');
-var TraceAgent = require('./src/trace-api.js');
-var pluginLoader = require('./src/trace-plugin-loader.js');
-var TraceWriter = require('./src/trace-writer.js');
+var constants = require('./src/constants'/*.js*/);
+var traceUtil = require('./src/util'/*.js*/);
+var TraceAgent = require('./src/trace-api'/*.js*/);
+var pluginLoader = require('./src/trace-plugin-loader'/*.js*/);
+var TraceWriter = require('./src/trace-writer'/*.js*/);
 
-var modulesLoadedBeforeTrace = [];
+var modulesLoadedBeforeTrace: any[] = [];
```

Comment:
could this be `string[]`?

Reply:
Done -- I do plan to more comprehensively add types in future PRs